prov/efa/test/efa_unit_test_srx.c

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) Amazon.com, Inc. or its affiliates.
+ * All rights reserved.
```

Comment:
nit indent

Reply:
What is the indent issue here?

Reply:
It's either on a single line
```
 * Copyright (c)  Amazon.com, Inc. or its affiliates. All rights reserved.
```

or indented like
```
 * Copyright (c)   Amazon.com, Inc. or its affiliates.
                   All rights reserved.
```

Reply:
I see, made it a single line